torch/_dynamo/config.py

```diff
@@ -380,7 +380,7 @@ def _get_optimize_ddp_mode():
 enable_cpp_guard_manager = True
 
 # Use C++ guard manger for symbolic shapes
-enable_cpp_symbolic_shape_guards = False
+enable_cpp_symbolic_shape_guards = not is_fbcode()
```

Comment:
@laithsakka the pr time benchmarks now depends on this value. Do you run those internally or in OSS only?

Reply:
I bumped up the results but once you rebase even if the test does not fail please bump the results since you know this will regress by 0.3% .
you can find the new suggested results in the logs of the test.

Reply:
Thanks @laithsakka for the information. The new suggested result is the exact same value as the one you bumped the results to, so I didn't bump it any further.